extension/src/debug.ts

```diff
@@ -405,6 +438,28 @@ export class ASDebugSession extends LoggingDebugSession
         }
     }
 
+    private resolvePathsToSourcePath(sourcePath : string, moduleName : string) : string
+    {
+        if (fs.existsSync(sourcePath))
+        {
+            return sourcePath;
+        }
+
+        let relativeFilename = moduleName.split(".").join("\\");
```

Comment:
vscode and this extension can run on more than just windows-based systems, so you should not assume that the path separator is backslash